src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+const getPartnerText = (person) => {
+  if (!person.isMarried) {
+    return 'I am not married';
+  }
+
+  if (person.sex === 'm') {
+    return `${person.partnerName} is my wife`;
+  }
+
+  return `${person.partnerName} is my husband`;
```

Comment:
```suggestion
  return `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`;
```
Only wife/husband text is different.